zebedee-cms/src/main/java/com/github/onsdigital/zebedee/model/Collections.java

```diff
@@ -577,6 +577,11 @@ public void writeContent(
             // create the file
             if (!collection.create(session.getEmail(), uri)) {
                 // file may be being edited in a different collection
+                Optional<Collection> otherCollection = zebedeeSupplier.get().checkForCollectionBlockingChange(uri);
+                if(otherCollection != null && otherCollection.isPresent()) {
```

Comment:
do you need the `!= null` I think optional should always return an value so you should be able to just do the `isPresent` check.

Reply:
I thought that, but then tests broke because it was throwing a `NullPointerException` in some cases

Reply:
Probably because the test isn't configured to return anything. You can use Optional.empty() to create an empty instance of the optional.